README.md

```diff
@@ -140,6 +140,15 @@ To run the container:
 
 `docker run --network=host -it thingd`
 
+#### Warning
+
+The docker environment has it own set of configuration files.
+
+You can edit them by using:
+
+`docker exec -it "$(docker ps -q --filter "ancestor=thingd")" /bin/sh`
```

Comment:
Nit: I'd rather give a more complete explanation (e.g.: run docker ps, indentify thingd container, exec -it [container_id]).